packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -54,13 +54,15 @@ export const Council = () => {
             { label: 'Period length', value: budget.refillPeriod, type: 'blocks' },
           ]}
         />
-        <MultiValueStat
-          title="Councilor Reward"
-          values={[
-            { label: 'Amount', value: reward.amount },
-            { label: 'Period length', value: reward.period, type: 'blocks' },
-          ]}
-        />
+        {reward?.amount && reward?.period && (
+          <MultiValueStat
+            title="Councilor Reward"
+            values={[
+              { label: 'Per Day', value: asBN(+reward.amount * +reward.period) },
+              { label: 'Per Week', value: asBN(7 * +reward.amount * +reward.period) },
```

Comment:
To avoid any risk of going over the max integer limit avoid casting these to numbers. Instead `reward.amount.mul(reward.period)` should work.

But also currently the reward period happens to be a day on mainnet but it could be changed in a runtime upgrade. So instead of relying on the current period length, the reward amount per day should be: `reward.amount.muln(Math.round(A_DAY / MILLISECONDS_PER_BLOCK))`.

Then:
```suggestion
              { label: 'Per Day', value: rewadPerDay },
              { label: 'Per Week', value: rewadPerDay.muln(7) },
```

Reply:
@traumschule

Reply:
Thanks! I almost committed your suggestion :)